bokehjs/src/coffee/common/plot.coffee

```diff
@@ -95,6 +95,8 @@ class PlotView extends ContinuumView
         el: @$(toolbar_selector)
       })
 
+      @mget('callback')?.execute(@model)
```

Comment:
I don't think this is in the right place. It's in the `if toolbar_location?` condition. It should be outside, unless I am missing something. I also think it should probably go after `@update_dataranges`